controllers/storagecluster/provider_server.go

```diff
@@ -28,11 +28,11 @@ type ocsProviderServer struct{}
 func (o *ocsProviderServer) ensureCreated(r *StorageClusterReconciler, instance *ocsv1.StorageCluster) error {
 
 	if !instance.Spec.AllowRemoteStorageConsumers {
-		r.Log.Info("nigoyal Spec.AllowRemoteStorageConsumers is disabled")
+		r.Log.Info("Spec.AllowRemoteStorageConsumers is disabled")
 		return o.ensureDeleted(r, instance)
 	}
 
-	r.Log.Info("nigoyal Spec.AllowRemoteStorageConsumers is enabled creating resources")
+	r.Log.Info("Spec.AllowRemoteStorageConsumers is enabled creating resources")
```

Comment:
```suggestion
	r.Log.Info("Spec.AllowRemoteStorageConsumers is enabled. Creating Provider API resources")
```

or specify which resources

Reply:
done